frontend/app/src/components/accounts/management/types/AddressAccountForm.vue

```diff
@@ -67,7 +68,7 @@ const save = async () => {
       const payload = entries.map(address => ({
         address,
         label: get(label),
-        tags: get(tags)
+        tags: emptyArrayToNull(get(tags))
```

Comment:
instead of doing it here can you use the `nonNullProperties` function when preparing to send the payload?